.github/workflows/pull_request.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: latest
+      - name: govulncheck
+        run: go install golang.org/x/vuln/cmd/govulncheck@latest; govulncheck ./...
```

Comment:
Would you please add this to the tools in `tools/tools.go` and `cd tools && go mod tidy`?

Reply:
Added. Thanks! Wasn't sure where to run `govulncheck` though. After unit-tests?